components/serverless/internal/controllers/serverless/function_reconcile_test.go

```diff
@@ -276,6 +277,9 @@ func TestFunctionReconciler_Reconcile_Scaling(t *testing.T) {
 }
 
 func TestFunctionReconciler_Reconcile(t *testing.T) {
+	// WARNING: This test function don't allow creating next test cases because of job limit in reconciler's configuration.
+	// New test cases should be created in separated test functions.
+	// TODO: Fix it
```

Comment:
Changin `testCfg.Build.MaxSimultaneousJobs = 10`may help the tests?

Reply:
Unfortunately nope. After increasing the `MaxSimultaneousJobs` even by 1 (to 6) is causing the test to never end... I think these tests need more effective refactoring because even adding one test case is a bit problematic